Formula/noti.rb

```diff
@@ -17,19 +16,20 @@ class Noti < Formula
   depends_on "go" => :build
 
   def install
-    mkdir_p "#{buildpath}/src/github.com/variadico"
-    ln_s buildpath, "#{buildpath}/src/github.com/variadico/noti"
     ENV["GOPATH"] = buildpath
-    cd "#{buildpath}/src/github.com/variadico/noti" do
-      if build.stable?
-        system "go", "build", "-o", "#{bin}/noti"
-      else
-        system "go", "build", "-o", "#{bin}/noti", "cmd/noti/main.go", "cmd/noti/osx.go"
-      end
+
+    # Move files to right location in GOPATH.
```

Comment:
Yeah, I did. We use it in quite a lot of places and I'd rather not encourage people to start documenting it in every single one. That's a problem we should really resolve with core documentation rather than in-formula notations.